tutorials/07-voting/README.md

```diff
@@ -0,0 +1,645 @@
+
+---
+We'll take you through these steps to get comfortable with the voting contracts.
+
+1. Deploy the contracts to the local blockchain emulator
+2. Create two user accounts
+3. Create voter accounts for these users
+4. Mint tokens to these accounts
+5. Create proposals for users to vote on
+6. Create `Ballot` resources for both voters
+7. Record and cast votes in the central voting contract
+8. Read the results of the vote
+
+## Setup the environment
+
+In this tutorial, we are introducing the [Flow CLI](https://developers.flow.com/tools/flow-cli/index), which allows us to run our code on a local blockchain emulator. 
+Please follow the link and install it on your computer. You should now be able to call it by simply entering `flow`, which will show you a list with all commands.
+The CLI also contains the blockchain emulator, please take a look at the [Flow CLI subsection](https://developers.flow.com/tools/flow-cli/start-emulator) for a brief overview, more details are covered in the [ReadMe](https://github.com/onflow/flow-emulator/#readme).
+
+Once installed, please download the project in the terminal by executing `git clone git@github.com:onflow/flow-playground-tutorials.git`, and then go to the project folder: `cd tutorials/08-voting`. 
+Once there, please execute this command which will create an initial configuration:
+
+```console
+flow init
+```
+
+Then, edit the configuration and add both the `contracts` and the `deployments` section, this is needed for the deployment of the project:
+
+```json:title=flow.json
+{
+    "contracts": {
+        "FungibleToken": "./contracts/FungibleToken.cdc",
+        "VotingTutorialAdministration": "./contracts/VotingTutorialAdministration.cdc",
+        "VotingTutorialGovernanceToken": "./contracts/VotingTutorialGovernanceToken.cdc"
+    },
+    //...networks, accounts...
+    },
+    "deployments": {
+        "emulator": {
+            "emulator-account": [
+                "FungibleToken",
+                "VotingTutorialGovernanceToken",
+                "VotingTutorialAdministration"
+            ]
+        }
+    }
+}
+```
+
+## Run the emulator
+
+Once the configuration is saved, you can start the emulator:
+
+```console
+flow emulator
+```
+
+## Deploy the project
+
+Open another terminal window (check that you remain in the project folder) and deploy the project:
+
+```console
+flow project deploy
+```
+
+All three contracts should be deployed now.
+FungibleToken is a Cadence standard, while `VotingTutorialGovernanceToken` and `VotingTutorialAdministration` are specific to this tutorial.
+`VotingTutorialGovernanceToken` is needed in order to vote, it's balance is determining the weight of the vote.
+`VotingTutorialAdministration` is used for administration of the whole voting process.
+
+`VotingTutorialGovernanceToken` contains the usual vault functionality of the FungibleToken contract and adds a history of voting weight that is updated on each transfer.
```

Comment:
We need a lot more than just a one sentence description of what the contract does. We need you to explain that most of it is identical to a regular fungible token implmentation, but then also walk through each important part that has been added for governance and explain what each of those is for, with code snippet examples to illustrate it

Reply:
I now added more explanation and code snippets.